### ISSUE TITLE
Splash Screen

### ISSUE THREAD

**Issue by mohandas-pai** [mohandas-pai/German-Lernen#2]
Got reviews from the users that a splash screen would be good for the app hence a small and quick splash screen would be amazing.

**Comment by silentwraith6** [comment]
I can work on this.

**Comment by raghavtalwar7** [comment]
i can also work on this

**Comment by mohandas-pai** [comment]
> I can work on this.

Hey, sure just to inform, the compileSdkVersion is 28, due to space constraint on my laptop i am not able to update, hope it isnt any issue.

**Comment by mohandas-pai** [comment]
> i can also work on this

hey, go ahead thanks a lot

**Comment by silentwraith6** [comment]
> > I can work on this.
> 
> Hey, sure just to inform, the compileSdkVersion is 28, due to space constraint on my laptop i am not able to update, hope it isnt any issue.

no issues

**Comment by raghavtalwar7** [comment]
> > I can work on this.
> 
> Hey, sure just to inform, the compileSdkVersion is 28, due to space constraint on my laptop i am not able to update, hope it isnt any issue.

No Problem

**Comment by mohandas-pai** [comment]
> i can also work on this

Just to inform i have another app that needs a splash screen maybe you can try that?

**Comment by mohandas-pai** [comment]
Not to sound greedy, but i do have another app that needs splash screen, i had put it yesterday!

**Comment by raghavtalwar7** [comment]
> Not to sound greedy, but i do have another app that needs splash screen, i had put it yesterday!

Sure no problem i'll work on that

**Comment by raghavtalwar7** [comment]
Where can i find it

**Comment by mohandas-pai** [comment]
> > Not to sound greedy, but i do have another app that needs splash screen, i had put it yesterday!
> 
> Sure no problem i'll work on that

Yes, that would help a lot, https://github.com/mohandas-pai/GBRB/issues/2
